README.md

```diff
@@ -1,104 +1,255 @@
+
+> [!NOTE] 
+> This repository contains code and documentation for the Elastic Open Web Crawler.
 Docker images are available for the crawler at [the Elastic Docker registry](https://www.docker.elastic.co/r/integrations/crawler).
 
 > [!IMPORTANT]
 > _The Open Crawler is currently in **beta**_.
 Beta features are subject to change and are not covered by the support SLA of generally available (GA) features.
 Elastic plans to promote this feature to GA in a future release.
 
-## Getting started
-
-This documentation outlines the following ways to run the Elastic Open Web Crawler:
-- [Simple Docker quickstart](#simple-docker-quickstart): Run a basic crawl with zero setup. No Elasticsearch instance required.
-- [Ingest into Elasticsearch](docs/ELASTICSEARCH.md): Configure the Open Crawler to connect to Elasticsearch and index crawl results.
-- [Developer guide](#developer-guide): Build and run Open Crawler from source, for developers who want to modify or extend the code.
-
 ### Version compatibility
 
 | Elasticsearch | Open Crawler       | Operating System |
 |---------------|--------------------|------------------|
 | `8.x`         | `v0.2.x`           | Linux, OSX       |
 | `9.x`         | `v0.2.1` and above | Linux, OSX       |
 
-### Simple Docker quickstart
 
-Let's scrape our first website using the Open Crawler running on Docker!
+## Getting started
+
+- [Quickstart](#quickstart): Use this hands-on guide to crawl a website's content into Elasticsearch using a simple configuration to get started.
+- [Documentation](#documentation): Learn how to configure advanced features and understand detailed options.
+- [Developer guide](#developer-guide): Learn how to build and run Open Crawler from source, for developers who want to modify or extend the code.
+
+### Quickstart
+
+Get from zero to crawling your website into Elasticsearch in just a few steps.
 
-The following commands will create a simple config file in your local directory, which will then be used by the Dockerized crawler to run a crawl.
-The results will be printed to your console, so no Elasticsearch setup is required for this step.
+#### Prerequisites
 
-Run the following commands from your terminal:
+- You'll need [Docker Desktop](https://docs.docker.com/desktop/) installed and running
+- You'll need a running Elasticsearch instance
+    - Start a free [Elastic Cloud Hosted or Serverless trial](https://www.elastic.co/cloud/cloud-trial-overview)
+    - [Get started locally](https://www.elastic.co/docs/solutions/search/run-elasticsearch-locally)
 
-``` bash
+> [!TIP]
+> If you haven't used Elasticsearch before, check out the [Elasticsearch basics quickstart](https://www.elastic.co/docs/solutions/search/elasticsearch-basics-quickstart) for a hands-on introduction to fundamental concepts.
+
+#### Step 1: Verify Docker setup
+
+First, let's test that the crawler works on your system by crawling a simple website and printing results to your terminal.
```

Comment:
I propose to keep the current format for this step where 
1. A user can run the crawler with one command - this has been added after being requested several times 
<img width="624" alt="image" src="https://github.com/user-attachments/assets/58461fbe-401b-4979-8abe-74cd505fe046" />

2. To help users understand what we're/they're doing, keep the existing explanation

> The following commands will create a simple config file in your local directory, which will then be used by the Dockerized crawler to run a crawl. The results will be printed to your console, so no Elasticsearch setup is required for this step.